packages/design-system/src/components/ChoiceList/ChoiceList.tsx

```diff
@@ -9,7 +9,7 @@ export type ChoiceListType = 'checkbox' | 'radio';
 export type ChoiceListErrorPlacement = 'top' | 'bottom';
 
 // Omit props that we override with values from the ChoiceList
-type OmitChoiceProp = 'inversed' | 'name' | 'onBlur' | 'onChange' | 'size' | 'type' | 'inputRef';
+type OmitChoiceProp = 'inversed' | 'name' | 'onBlur' | 'onChange' | 'size' | 'type';
```

Comment:
Oh, I didn't realize that we were omitting this before. It being part of the expected props was the basis of my justification for calling this a bugfix 😆